Slack/legos/slack_create_channel_invite_users/slack_create_channel_invite_users.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+##
+# Copyright (c) 2023 unSkript, Inc
+# All rights reserved.
+##
+import pprint
+from typing import List
+from pydantic import BaseModel, Field
+from slack_sdk import WebClient
+## note: Your Slack App will need the files:write scope.
+# Your Bot will also need to be a member of the channel
+
+
+
+
+class InputSchema(BaseModel):
+    channel: str = Field(..., description='Name of slack channel.', title='Channel')
+    user_list: List = Field(
+        ...,
+        description='List of users to invite to the new channel.',
```

Comment:
Give an example: is it `@abhishek` or `abhishek`?

Reply:
slackIDs.

Example added